wagtail_extensions/static/wagtail_extensions/scss/components/carousel.scss

```diff
@@ -14,4 +14,27 @@
         padding-bottom: 40px;
         background: rgba(0, 0, 0, 0.7);
     }
+
+    &.with-thumbnails {
+        margin-bottom: 115px;
+    }
+
+    .carousel-thumbnails {
+        bottom: -100px;
+        margin-left: 0;
+        margin-right: 0;
+        padding: 15px;
+        background: #000;
+
+        li {
+            width: auto !important;
```

Comment:
`important` makes it very hard to override this styling at project level. It is possible to achieve this without using `important`.